papaparse.js

```diff
@@ -1516,6 +1516,7 @@ License: MIT
 
 			var nextDelim = input.indexOf(delim, cursor);
 			var nextNewline = input.indexOf(newline, cursor);
+			// Pre-compile the regex for escaped quotes for performance
```

Comment:
I do not see any difference despite the comment. So I understand nothing is changed here. Rirgh? Or what is your change?